ioctl/newcmd/account/accountethaddr_test.go

```diff
@@ -25,7 +25,6 @@ func TestNewAccountEthAddr(t *testing.T) {
 	defer ctrl.Finish()
 	client := mock_ioctlclient.NewMockClient(ctrl)
 	client.EXPECT().SelectTranslation(gomock.Any()).Return("", config.English).AnyTimes()
-	client.EXPECT().PrintInfo(gomock.Any()).Times(2)
```

Comment:
The result of L32 can be checked. So check other possible places which could be checked.